websites/Y/YouTube Music/presence.ts

```diff
@@ -194,7 +194,8 @@ presence.on("UpdateData", async () => {
 
 			presenceData.largeImageKey =
 				document.querySelector<HTMLImageElement>("#thumbnail img").src;
-			presenceData.smallImageKey = "ytm_lg";
+			presenceData.smallImageKey =
+				"https://cdn.discordapp.com/app-assets/463151177836658699/463151407512682502.png";
```

Comment:
Assets should be 512x512px.
```suggestion
				"https://cdn.discordapp.com/app-assets/463151177836658699/463151407512682502.png?size=512";
```